docs/developer-docs/latest/setup-deployment-guides/installation/digitalocean-one-click.md

```diff
@@ -5,11 +5,11 @@ description: Quickly deploy a Strapi application on DigitalOcean by simply using
 
 # DigitalOcean One-click
 
-::: caution
- The one-click install droplet is currently missing from DigitalOcean's marketplace. We are actively working with them on this issue and recommend you to use another installation method in the meantime.
- :::
+::: warning
```

Comment:
The `warning` callout is the highest level of "danger" in Strapi docs. We only use it for "things that will definitely break your app". Are you sure this is the required level here?